docs/user/agent/mce-integration.md

```diff
@@ -3,6 +3,106 @@
+    architectures:
+      - "amd64"
+    channels:
+      - name: stable-4.11
+        type: ocp
+  additionalImages:
+    - name: registry.redhat.io/ubi8/ubi:latest
+  operators:
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.11
+      packages:
+        - name: multicluster-engine
+        - name: local-storage-operator
+```
+
+With this file you will be able to use this command to mirror the OCP release and the MCE and LSO operators:
+
+```oc mirror --dest-skip-tls --config ocp-mce-imageset.yaml docker://<your-local-registry-dns-name>:<your-local-registry-port>```
+
+**Q: Why do I need the LSO operator?** A: The MCE operator is a large package and comes packaged with the infrastructure-operator as well. The infrastructure-operator needs a local storage volume to function.
+
+### Update the mirror configurations in install-config.yaml
+
+In your __install-config.yaml__ you will need to update the registry and the certificate. The two configurations to update are `imageContentSources` and `additionalTrustBundle`.
+The following file snippet shows how to update the `imageContentSources` this for OCP and the MCE+LSO operators.
+
+```
+imageContentSources:
+  - source: "quay.io/openshift-release-dev/ocp-release"
+    mirrors:
+      - "<your-local-registry-dns-name>:<your-local-registry-port>/openshift/release-images"
+  - source: "quay.io/openshift-release-dev/ocp-v4.0-art-dev"
+    mirrors:
+      - "<your-local-registry-dns-name>:<your-local-registry-port>/openshift/release"
+  - source: "registry.redhat.io/ubi8"
+    mirrors:
+      - "<your-local-registry-dns-name>:<your-local-registry-port>/ubi8"
+  - source: "registry.redhat.io/multicluster-engine"
+    mirrors:
+      - "<your-local-registry-dns-name>:<your-local-registry-port>/multicluster-engine"
+  - source: "registry.redhat.io/rhel8"
+    mirrors:
+      - "<your-local-registry-dns-name>:<your-local-registry-port>/rhel8"
+  - source: "registry.redhat.io/redhat"
+    mirrors:
+      - "<your-local-registry-dns-name>:<your-local-registry-port>/redhat"
+```
+
+**Note:** This procedure with `oc mirror` can be used to mirror any of the operators in the Red Hat Operator Indexes. After running `oc mirror` there will be a folder called `oc-mirror-workspace` with several outputs. There will be a file called __imageContentSourcePolicy.yaml__ in the most recent results folder that will identify all the mirrors you need for OCP and your selected operators.
+
+You will also need to make sure your certificate is present in the `additionalTrustBundle` field.
+This should look like something like this in your __install-config.yaml__:
+
+```
+additionalTrustBundle: |
+  -----BEGIN CERTIFICATE-----
+  .
+  .
+  .
+  .
+  .
+  .
+  -----END CERTIFICATE-----
+```
+
+Now you can generate the cluster manifests with the command:
+
+`openshift-install agent create cluster-manifests`
+
+This will update the cluster manifests to include a `mirror` folder with your mirror configuration.
```

Comment:
There's no actual need for this step is there? I think it's fine to go straight to `create image`.

Reply:
🤔 Will test this scenario tomorrow.

Reply:
It's just documentation updates but before pushing the changes up I'm actually testing it manually. 😅